README.md

```diff
@@ -99,127 +99,142 @@ val config: Try[YourConfClass] = loadConfig[YourConfClass]
 
 Currently supported types for fields are:
 - `String`, `Boolean`, `Double` (standard
-and percentage format ending with `%`), `Float` (also supporting percentage),
-`Int`, `Long`, `Short`, `URL`, `Duration`, `FiniteDuration`
-- all collections implementing the `TraversableOnce` trait where the elements type
-is in this list
-- `Option` for optional values, i.e. value that can or cannot be in the configuration
-- `Map` with `String` keys and any value type that is in this list
-- everything in [`java.time`](https://docs.oracle.com/javase/8/docs/api/java/time/package-summary.html).
- Because different formats are supported, converters for types that support multiple formats must be configured
- before they can be used (see [Configurable converters](#configurable-converters))
-- typesafe `ConfigValue`, `ConfigObject` and `ConfigList`
-- case classes
-- sealed families of case classes (ADTs)
+  and percentage format ending with `%`), `Float` (also supporting percentage),
+  `Int`, `Long`, `Short`, `URL`, `Duration`, `FiniteDuration`;
+- all collections implementing the `TraversableOnce` trait where the type of
+  the elements is in this list;
+- `Option` for optional values, i.e. values that can or cannot be in the configuration;
+- `Map` with `String` keys and any value type that is in this list;
+- everything in [`java.time`](https://docs.oracle.com/javase/8/docs/api/java/time/package-summary.html) (must be
+  configured first - see [Configurable converters](#configurable-converters));
+- Typesafe `ConfigValue`, `ConfigObject` and `ConfigList`;
+- case classes;
+- sealed families of case classes (ADTs).
 
 An almost comprehensive example is:
 
 ```scala
-> import pureconfig.loadConfig
-> import com.typesafe.config.ConfigFactory.parseString
-> sealed trait MyAdt
-> case class AdtA(a: String) extends MyAdt
-> case class AdtB(b: Int) extends MyAdt
-> case class MyClass(int: Int, adt: MyAdt, list: List[Double], map: Map[String, String], option: Option[String])
-> val conf = parseString("""{ "int": 1, "adt": { "type": "adtb", "b": 1 }, "list":["1", "20%"], "map": { "key": "value" } }""")
-> loadConfig[MyClass](conf)
-res0: util.Try[MyClass] = Success(MyClass(1,AdtB(1),List(1.0, 0.2),Map(key -> value),None))
+import com.typesafe.config.ConfigFactory.parseString
```

Comment:
Regarding self-contained examples, I opted to treat all examples as a single console session, which means I didn't repeat `import` statements made before in the README. That's why I removed some of them and that's also why I used `parseString` instead of `ConfigFactory.parseString`. But I'm ok either way; we can "re-import" everything in each example, it makes examples more self-contained :)